simpleflow/task.py

```diff
@@ -155,6 +161,12 @@ def name(self):
     def execute(self):
         pass
 
+    def propagate_attribute(self, attr, val):
+        """
+        No propagation.
```

Comment:
(minor) I'd complete the docstring with "We've not established a policy yet for child workflows." (Shocking, it's still WIP :slightly_smiling_face:)